lib/wootric/index.js

```diff
@@ -0,0 +1,91 @@
+var integration = require('analytics.js-integration');
+
+/**
+ * Expose `Wootric` integration.
+ */
+
+var Wootric = module.exports = integration('Wootric')
+  .assumesPageview()
+    .option('account_token', '')
+    .global('wootricSettings')
+    .global('wootric_survey_immediately')
+    .global('wootric')
+    //The integration object
+    .global('_wootric')
+    .tag('library', '<script src="http://d27j601g4x0gd5.cloudfront.net/segmentioSnippet.js"></script>')
```

Comment:
Can this script run over https? If yes, remove the protocol from the front (should end up looking like this: `//d27j601g4x0gd5.cloudfront.net/segmentioSnippet.js`)